run_windows.go

```diff
@@ -0,0 +1,527 @@
+package pty
+
+import (
+	"context"
+	"errors"
+	"io"
+	"os"
+	"os/exec"
+	"runtime"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+type startupInfoEx struct {
+	startupInfo syscall.StartupInfo
+	lpAttrList  syscall.Handle
+}
+
+const (
+	_PROC_THREAD_ATTRIBUTE_PSEUDOCONSOLE = 0x00020016
```

Comment:
Could you add a comment about what this is / where it comes from and why we need it?

Reply:
Well, I could not find an official page for this value, but the terminal sample app from microsoft suggests this is correct.

FYI: https://github.com/microsoft/terminal/blob/89ca2ae05f7b47bebaec7f6a8ff8538770851af3/samples/ConPTY/GUIConsole/GUIConsole.ConPTY/Native/PseudoConsoleApi.cs#L12